cmd/main.go

```diff
@@ -17,11 +17,59 @@
 package main
 
 import (
+	"flag"
+	"fmt"
+	"log"
 	"os"
+	"os/exec"
+
+	"golang.org/x/exp/slices"
 
 	"github.com/milvus-io/milvus/cmd/milvus"
+	"github.com/milvus-io/milvus/internal/util/paramtable"
+	"github.com/milvus-io/milvus/internal/util/sessionutil"
 )
 
 func main() {
-	milvus.RunMilvus(os.Args)
+	idx := slices.Index(os.Args, "--run-with-subprocess")
+
+	// execute command as a subprocess if the command contains "--run-with-subprocess"
+	if idx > 0 {
+		args := slices.Delete(os.Args, idx, idx+1)
+		fmt.Println("run subprocess with cmd:", args)
+
+		/* #nosec G204 */
+		cmd := exec.Command(args[0], args[1:]...)
+
+		cmd.Stdout = os.Stdout
+		cmd.Stderr = os.Stderr
+
+		// No need to extra wait for the process
+		err := cmd.Run()
```

Comment:
this will block until server crash?

Reply:
Yes,   it will block until the server exits with normal or abnormal status.